[2021-05-09 18:01:21,413] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_4.python_push_into_target 2021-05-08T05:53:45.288265+00:00 [queued]>
[2021-05-09 18:01:21,432] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_4.python_push_into_target 2021-05-08T05:53:45.288265+00:00 [queued]>
[2021-05-09 18:01:21,432] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-09 18:01:21,432] {taskinstance.py:1069} INFO - Starting attempt 1 of 3
[2021-05-09 18:01:21,432] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-09 18:01:21,587] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): python_push_into_target> on 2021-05-08T05:53:45.288265+00:00
[2021-05-09 18:01:21,590] {standard_task_runner.py:52} INFO - Started process 6505 to run task
[2021-05-09 18:01:21,595] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_4', 'python_push_into_target', '2021-05-08T05:53:45.288265+00:00', '--job-id', '127', '--pool', 'default_pool', '--raw', '--subdir', '/home/navneetsajwan/airflow/dags/dag__4.py', '--cfg-path', '/tmp/tmp1l8zfdzh', '--error-file', '/tmp/tmp6lxz6_1f']
[2021-05-09 18:01:21,596] {standard_task_runner.py:77} INFO - Job 127: Subtask python_push_into_target
[2021-05-09 18:01:21,714] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_4.python_push_into_target 2021-05-08T05:53:45.288265+00:00 [running]> on host navneetsajwan-Lenovo-G50-80
[2021-05-09 18:01:21,837] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=navneet@metaorigins.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dag_4
AIRFLOW_CTX_TASK_ID=python_push_into_target
AIRFLOW_CTX_EXECUTION_DATE=2021-05-08T05:53:45.288265+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-05-08T05:53:45.288265+00:00
[2021-05-09 18:01:22,896] {logging_mixin.py:104} INFO - push_test
[2021-05-09 18:01:22,896] {python.py:118} INFO - Done. Returned value was: None
[2021-05-09 18:01:22,903] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=dag_4, task_id=python_push_into_target, execution_date=20210508T055345, start_date=20210509T123121, end_date=20210509T123122
[2021-05-09 18:01:23,245] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-05-09 18:01:23,293] {local_task_job.py:146} INFO - Task exited with return code 0
